hugo/layouts/partials/teaser.html

```diff
@@ -9,6 +9,7 @@ <h2 class="teaser__title">{{ .Title }}</h2>
     <p class="teaser__meta">{{ .Date.Format ($.Param "time_format") }}</p>
     <p class="teaser__excerpt">{{ .Summary | truncate 300 }}</p>
     <div class="link teaser__readmore">
+        hoi
```

Comment:
There's still some test text here

Reply:
This will need to be addressed in a follow-up

Reply:
Made a linear issue: https://linear.app/usmedia/issue/CUE-277